concourse/replicator.py

```diff
@@ -157,8 +178,19 @@ def _render(self, definition_descriptor):
             'target_team': definition_descriptor.concourse_target_team,
             'secret_cfg': definition_descriptor.secret_cfg,
             'job_mapping': definition_descriptor.job_mapping,
+            'render_origin': self.render_origin,
+            'cc_utils_version': _cc_utils_version(),
+            'ref_sha': definition_descriptor.ref_sha,
         }
 
+        # also pass pipeline name if this was rendered by a replication job. Will be printed
+        # in the meta-step later
+        if (
```

Comment:
why not simply always add PIPELINE_NAME or `<none>`?

Reply:
vanity